app/javascript/packages/document-capture/components/file-input.jsx

```diff
@@ -165,6 +177,11 @@ const FileInput = forwardRef((props, ref) => {
           {shownErrorMessage}
         </span>
       )}
+      {isUpdated && !shownErrorMessage && (
+        <span className="usa-success-message" role="alert">
+          {fileUpdatedText ?? t('forms.file_input.file_updated')}
```

Comment:
Why have the default translated text? Looks like in all our use cases we pass in a value?

Reply:
> Why have the default translated text? Looks like in all our use cases we pass in a value?

Short answer: Yes

Long answer: I've been eyeing this component as one to move to the `@18f/identity-components` workspace package to make reusable, and as such avoid making assumptions about how this "updated" text would be presented. This all related to ongoing UX discussion around a potential [design system component](https://design.login.gov/components/file-uploads/).

It may be a bit premature. I'd also been thinking about a compromise solution which is to have the default be a hard-coded English text, or to allow the component to handle `fileUpdatedText` as optional (for example, not show the text unless the value is given).

Reply:
I'll plan to keep this as-is.